test/sender.js

```diff
@@ -54,6 +54,25 @@ describe('Event sender', () => {
       })
     })
   })
+
+  it('sends using RFC 5424', done => {
+    const server = new TestServer(insecureOptions, () => {
+      const sender = senderLib.create({
+        ...insecureOptions,
+        rfc5424: true,
+      });
+      sender.on('error', done)
+      sender.send(messageString)
+      server.waitFor('data', data => {
+        //console.log('data %s', data);
+        String(data).should.containEql(messageString)
+        String(data).should.containEql(year)
```

Comment:
How is this testing specifically the RFC5424 fotmat? Other tests also check for `messageString` and `year` but they don't use RFC5424 format. I would suggest to check something specific of this format.

Reply:
Good idea! Done in commit c3fac47.